start/resources/singularity-containers.md

```diff
@@ -88,170 +27,153 @@ tools and libraries. The images are are:
 
-### FATAL: kernel too old ###
+    output = out
+    error = err
+    log = log
 
-If you get a *FATAL: kernel too old* error, it means that the glibc version in the
-image is too new for the kernel on the host. You can work around this problem by
-specifying the minimum host kernel. For example, if you want to run the Ubuntu 18.04
-image, specfy a minimum host kernel of 3.10.0, formatted as 31000
-(major * 10000 + minor * 100 + patch):
+    queue
 
-    Requirements = HAS_SINGULARITY == True && OSG_HOST_KERNEL_VERSION >= 31000
+## Using Custom Singularity Images
 
+OSG Connect provides tooling for users to create, publish and load custom images.
+This is useful if your job requires some very specific software setup. The general 
+process goes like this: 
 
-## Custom Images
+1. Create your own custom container image using **Docker** and push it to Docker Hub. 
+2. Add your Docker image to the Open Science Grid image repository. 
+3. Use the container image in jobs. 
 
-OSG Connect provides tooling for users to create, publish and load custom images.
-This is useful if your job requires some very specific software setup.
+We will expand on each of these steps below. 
 
-### Creating a Custom Image
+### Creating a Container
 
-If you want to use an image you have created yourself, the image
-should be defined as a Docker image and published in the [Docker
+If you want to use an container image you have created yourself, the image
+should be defined as a Docker image and published in [Docker
 Hub](https://hub.docker.com/). The reason we use Docker as a source
 image repository is that it allows us to easily import the images into
-our own distribution system (see below). To get started, create a Docker
-user, sign in to the hub, and create a new repository. You will end up
-with an identifier of the `namespace/repository_name` format.
-
-There are two main methods, we recommend to generating your own
-custom image:
-
-1. Editing the `Dockerfile`
-2. Editing the default image using local Docker
-
-which are described below. Once you have created your custom image, 
-you will need to register the image as described in the next section. 
-If you prefer, you can base you image on images not already published 
-by OSG, but if you do this, we recommend that you as one of the steps 
-create the `/cvmfs` directory. This will enable the container to access 
-tools and data published on `/cvmfs`. If you do not want `/cvmfs` mounted 
-in the container, please add `+SingularityBindCVMFS = False` to your job submit file.
-
-#### Editing the `Dockerfile`
+our own distribution system (see below). 
 
-To create an image locally using a `Dockerfile`, 
-first download the `Dockerfile` for your desired flavor from 
-[OSG GitHub repository](https://github.com/opensciencegrid), 
-see table above. For example, if you want to base the image on our Ubuntu
-Xenial image, 
+See [this page][container-howto] for how to create a Docker image on your own computer and 
```

Comment:
This should be `[this page](container-how-to)`

Reply:
I'm using reference style links.